tarteaucitron.services.js

```diff
@@ -2397,6 +2397,52 @@ tarteaucitron.services.purechat = {
     }
 };
 
+// Intercom
+tarteaucitron.services.intercomChat = {
+    "key": "intercomChat",
+    "type": "support",
+    "name": "Intercom",
+    "uri": "https://www.intercom.com/",
+    "needConsent": true,
+    "cookies": [
+        "intercom-id-" + tarteaucitron.user.ingercomKey,
+        "intercom-session-" + tarteaucitron.user.ingercomKey,
+    ],
+    "readmoreLink": "https://www.intercom.com/legal/privacy",
+    "js": function () {
+        window.intercomSettings = {
+            app_id: tarteaucitron.user.ingercomKey,
+        };
+
+        var w = window;
+        var ic = w.Intercom;
+        if (typeof ic === "function") {
+            ic("reattach_activator");
+            ic("update", w.intercomSettings);
+        } else {
+            var i = function () {
+                i.c(arguments);
+            };
+            i.q = [];
+            i.c = function (args) {
+                i.q.push(args);
+            };
+            w.Intercom = i;
+            tarteaucitron.addScript(
+                "https://widget.intercom.io/widget/" + tarteaucitron.user.intercomKey,
+                "",
+                intercomChatEnable(),
+            );
+        }
+    },
+    "fallback": function () {
+        "use strict";
+        var id = "intercomChat";
+        tarteaucitron.fallback(["intercom-chat"], tarteaucitron.engage(id));
+        intercomChatDisable()
```

Comment:
This function will thrown an error because the intercom script will not be loaded.

Reply:
@AmauriC sorry for the dalay. 

I've corrected this. Could you check again please ?

Reply:
Hey,
I still see the `intercomChatDisable()` function 🧐